### ISSUE TITLE
"Getting Started" page for Visitors is incorrect

### ISSUE THREAD

**Issue by jafrahm** [ResearchComputing/website#87]
https://www.rc.colorado.edu/support/getting-started/sponsored-user.html

Visitors and external collaborators cannot self-create accounts and must email rc-help with their sponsored IdentiKey and other relevant information to get an account.  They also cannot request a Startup allocation and need to be added to a Project allocation by a CU PI.

We do not usually need much info from the user, the SN case for sponsorship should have it, but we can ask for:
name
IdentiKey username
Best contact email
CU Boulder faculty they are working with

**Comment by anderbubble** [comment]
No-one self-creates accounts: these are all account requests. At least, I thought that's how it worked.

If an external collaborator has a sponsored account and, thus, an IdentiKey username and password already, how do we distinguish them from a CU user that is ostensibly permitted to request an account on their own, and to request a startup allocation? (That is, "How is it already being done today?" Not "How could we do it in the future?")
